readthedocs/core/static-src/core/js/readthedocs-doc-embed.js

```diff
@@ -5,12 +5,43 @@ const footer = require('./doc-embed/footer.js');
 const sphinx = require('./doc-embed/sphinx');
 const search = require('./doc-embed/search');
 const { domReady } = require('./doc-embed/utils');
+const rtddata = require('./doc-embed/rtd-data');
+
+/*
+ * Inject JQuery if isn't present already.
+ *
+ * Parts of this script rely on JQuery (mainly the flyout menu injection),
+ * since Sphinx no longer includes it, and other tools may not include it,
+ * we must inject it if isn't found before executing our script.
+*/
+function injectJQuery(init) {
+    if (window.jQuery) {
+        init()
+        return
+    }
+    console.debug("JQuery not found. Injecting.");
+    let rtd = rtddata.get();
+    let script = document.createElement("script");
+    script.type = "text/javascript";
+    script.src = rtd.proxied_static_path + "vendor/jquery.js";
+    script.onload = function () {
+        // Set jQuery to its expected globals.
+        window.$ = require("jquery");
+        window.jQuery = window.$;
```

Comment:
Not really sure if this is the way of doing it... but just adding the script from vendor/jquery.js doesn't add the global variables :/ /cc @agjohnson

Reply:
Yup, this is normal. Relying on every module to be a global is an antipattern that doesn't work well with various forms of module importing -- commonJS, AMD, ES6 module imports, etc. Modern JS would tend to use jQuery as a module instead.

Your change is about what I'd do.